tests/wpt/web-platform-tests/html/semantics/embedded-content/media-elements/event_canplay.html

```diff
@@ -14,25 +14,27 @@
   </video>
   <div id="log"></div>
   <script>
-test(function() {
+(function() {
```

Comment:
You probably want to leave the outer test() so things fail sanely if the .src assignment throws or something.